src/components/Card.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 
-const Card = () => {
+const Card = props => {
```

Comment:
why don't you destructure the props...?
`const Card = ({title, children})  => {`